src/components/fields/LineSelectors.js

```diff
@@ -93,13 +82,7 @@ class LineSelector extends Component {
 
   render() {
     return (
-      <Dropdown
-        {...this.props}
-        options={this.options}
-        valueRenderer={styledRenderer}
```

Comment:
again.. needed

Reply:
This particular spot - do we really need them here? It's a line selector, it's not a data source thing, we don't pass things here down from EditorControls. I don't think we need these props here.